packages/webpack/src/buildtime/wrapper.js

```diff
@@ -33,13 +41,26 @@ exports.wrapper = function wrapper({
   runtimeKit,
   evalKitFunctionName,
   runChecks = true,
+  runtimeFlags = {},
 }) {
+  const runtimeKitArray = Array.from(runtimeKit)
   // validateSource(source);
 
   // No AST used in these transforms, so string cmparison should indicate if anything was changed.
   const sesCompatibleSource = applySourceTransforms(source)
   const sourceChanged = source !== sesCompatibleSource
 
+  // This adds support for mapping `this` to `exports` or `module.exports` if webpack detected it's necessary
+  let optionalBinding = ''
+
+  if (runtimeFlags.thisAsExports) {
```

Comment:
I think this is great, but it took me a while to realize that `exports` and `module` are hardcoded statements coming from webpack, I feel like if there's a way to grab them and use them instead, it would make this code more clear (if possible):

```javascript
const {exportsArgument, moduleArgument} = Webpack.consts.arguments;
```

```javascript
if runtimeKitArray.includes(exportsArgument) 
if runtimeKitArray.includes(moduleArgument) 
```